raccoochat/RaccooChatClient.py

```diff
@@ -24,43 +25,49 @@ def __init__(self, host, port):
         self.transport.open()
 
     def print_messages(self, message_array):
-        for msg in message_array:
-            shift = 10 - len(msg.userName)
-            print '[%s] %s %s : %s ' % (msg.time, ' ' * shift, msg.userName, msg.textMessage)
+        for message in message_array:
+            shift = 10 - len(message.userName)
+            print '[%s] %s %s : %s ' % (message.time, ' ' * shift, message.userName, message.textMessage)
+
+    def print_private_messages(self, message_array):
+        for message in message_array:
+            shift = 10 - len(message.userName)
```

Comment:
i am working now on keeping users and passwords in a file.
This function will be changed.